### PR TITLE
Allow setting a min TLP and default TLP configurations

### LINKED ISSUE #599: A configuration property to restrict TLP
Add 2 new configuration properties:

- `allowed_tlps` : allow clients to POST using only those TLPs
- `default-tlp`: the default tlp value for newly posted docs when none is set

### PR DESCRIPTION
**Pull request by gbuisson**
Allow configuring default, and minimum TLP level an API customer can submit documents with.

closes #599

### CODE REVIEW

File: src/ctia/domain/entities.clj [hunk 1]
```diff
@@ -2,6 +2,9 @@
   (:require
    [clj-momo.lib.time :as time]
    [ctia.properties :refer [get-http-show]]
+   [ctia.domain.access-control
+    :refer [properties-default-tlp
+            allowed-tlps]]
    [ctim.domain.id :as id]
    [ring.util.http-response :as http-response]
    [schema.core :as s]
```
Comment (msprunck) [inline src/ctia/domain/entities.clj]: You could remove the reference to `default-tlp` in the `:require`

File: src/ctia/properties.clj [hunk 2]
```diff
@@ -82,6 +78,9 @@
                       "ctia.hook.redis.enabled" s/Bool
                       "ctia.hook.redismq.enabled" s/Bool})
 
+   (st/required-keys {"ctia.access-control.min-tlp" s/Str
+                      "ctia.access-control.default-tlp" s/Str})
```
Comment (msprunck) [inline src/ctia/properties.clj]: You could use the TLP schema from CTIM

File: src/ctia/domain/access_control.clj [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
   ["white" "green"])
 
+(def tlps ["white" "green" "amber" "red"])
+
+(defn properties-default-tlp []
+  (or (:default-tlp (get-access-control))
+      csc/default-tlp))
+
+(defn allowed-tlps []
+  (let [min-tlp (:min-tlp (get-access-control))]
```
[the remark this answers is not in the document]
Comment (saintx) [reply src/ctia/domain/access_control.clj]: Agreed.

File: src/ctia/properties.clj (hunk 2 above)
Comment (saintx) [reply src/ctia/properties.clj]: Also agreed.  Let's reuse the schema we've got.

**Pull request merged**